Makefile

```diff
@@ -21,7 +21,7 @@ BUILD_OUTPUT_DIRECTORY = $(BUILD_DIRECTORY)/out
 # ---------------------------------------------------------------------
 
 ELECTRON_VERSION = $(shell jq -r '.devDependencies["electron"]' package.json)
-NODE_VERSION = 6.1.0
+NODE_VERSION = $(shell node --version)
```

Comment:
Does that mean that the node-static-entry-point thingy that @jviotti made would also need to be updated to use node8?  (disclaimer: I've not looked at how that tool works)